www/app/Views/login.html

```diff
@@ -0,0 +1,399 @@
+            background: transparent;
+        }
+
+
+        /* Ripples circle */
+
+        .ripplesCircle {
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            -webkit-transform: translate(-50%, -50%);
+            transform: translate(-50%, -50%);
+            opacity: 0;
+            width: 0;
+            height: 0;
+            border-radius: 50%;
+            background: rgba(255, 255, 255, 0.25);
+        }
+
+        .ripples.is-active .ripplesCircle {
+            -webkit-animation: ripples .4s ease-in;
+            animation: ripples .4s ease-in;
+        }
+
+
+        /* Ripples animation */
+
+        @-webkit-keyframes ripples {
+            0% {
+                opacity: 0;
+            }
+
+            25% {
+                opacity: 1;
+            }
+
+            100% {
+                width: 200%;
+                padding-bottom: 200%;
+                opacity: 0;
+            }
+        }
+
+        @keyframes ripples {
+            0% {
+                opacity: 0;
+            }
+
+            25% {
+                opacity: 1;
+            }
+
+            100% {
+                width: 200%;
+                padding-bottom: 200%;
+                opacity: 0;
+            }
+        }
+
+        footer {
+            text-align: center;
+        }
+
+        footer p {
+            color: #888;
+            font-size: 13px;
+            letter-spacing: .4px;
+        }
+
+        footer a {
+            color: #4a89dc;
+            text-decoration: none;
+            transition: all .2s ease;
+        }
+
+        footer a:hover {
+            color: #666;
+            text-decoration: underline;
+        }
+
+        footer img {
+            width: 80px;
+            transition: all .2s ease;
+        }
+
+        footer img:hover {
+            opacity: .83;
+        }
+
+        footer img:focus,
+        footer a:focus {
+            outline: none;
+        }
+    </style>
+    <script>
+        window.console = window.console || function (t) { };
+    </script>
+    <script>
+        if (document.location.search.match(/type=embed/gi)) {
+            window.parent.postMessage("resize", "*");
+        }
+    </script>
+</head>
+
+<body translate="no">
+    <hgroup>
+        <h1>Template Login</h1>
+    </hgroup>
+    <form method="POST" action="{{ baseUrl }}/connect" >
+        <div class="group">
+            <input type="text"><span class="highlight"></span><span class="bar"></span>
+            <label>E-mail</label>
+        </div>
+        <div class="group">
+            <input type="email"><span class="highlight"></span><span class="bar"></span>
+            <label>Senha</label>
+        </div>
+        <button type="button" class="button buttonBlue">Entrar
+            <div class="ripples buttonRipples"><span class="ripplesCircle"></span></div>
+        </button>
+    </form>
+    <footer><a href="https://www.dprimeira.com.br/" target="_blank"><img
```

Comment:
Replace this footer. The company is LyseonTech.

Reply:
ok